datetime_tz/win32tz_map.py

```diff
@@ -0,0 +1,110 @@
+'''Map between Windows and Olson timezones taken from http://www.unicode.org/repos/cldr/trunk/common/supplemental/windowsZones.xml
```

Comment:
Is there any reason we shouldn't just generate it fresh rather than updating an existing file?